src/applications/gi/reducers/autocomplete.js

```diff
@@ -61,7 +61,7 @@ export default function(state = INITIAL_STATE, action) {
     case SEARCH_STARTED:
       return {
         ...state,
-        searchTerm: action.query.name,
+        searchTerm: action.query.name ? action.query.name : '',
```

Comment:
You can make use of the `get` utility from `src/platform/utilities/data/get`.

```
searchTerm: get('query.name', action, ''),
```

That utility's useful if there are multiple segments in an object path that you can't guarantee to be valid. I suppose in this case, you know `query` will be at least an empty object, so maybe this can work too:

```
searchTerm: action.query.name || '',
```

Reply:
changed